src/main/resources/default/taglib/t/lookupValue.html.pasta

```diff
@@ -6,6 +6,7 @@
 <i:arg name="help" type="String" default="@i18n(helpKey)"/>
 <i:arg name="optional" type="boolean" default="false"/>
 <i:arg name="readonly" type="boolean" default="false"/>
+<i:arg name="showAddon" type="boolean" default="true"/>
```

Comment:
maybe call this showSelectionButton or the like - addon is a broad term used by the single select itself, but here we know what it is used for